### PR TITLE
Do not crash if Home wiki page is deleted, just allow user to create another Home

### PR DESCRIPTION
**Pull request by linquize**
Do not crash if Home wiki page is deleted, just allow user to create another Home

### CODE REVIEW

File: routers/repo/wiki.go [hunk 1]
```diff
@@ -75,6 +78,16 @@ func CreateWikiPage(ctx *middleware.Context) {
 		ctx.Data["PageTitle"] = "Home"
 	}
 
+	has, er := models.HasWikiPage(wr, "Home")
```
Comment (tboerger) [inline routers/repo/wiki.go]: This should be err :)

Comment (linquize) [inline routers/repo/wiki.go]: done

File: routers/repo/wiki.go [hunk 2]
```diff
@@ -34,12 +34,15 @@ func Wiki(ctx *middleware.Context) {
 	canWrite, err := models.HasAccess(ctx.User, wr, models.ACCESS_MODE_WRITE)
 	if err != nil {
 		ctx.Handle(500, "wiki.Wiki", err)
+		return
 	}
 
 	p, err := models.GetWikiPage(wr, "home")
 	if err != nil {
-		ctx.Handle(500, "wiki.Wiki", err)
+		ctx.HTML(200, WIKI_EMPTY)
```
Comment (lunny) [inline routers/repo/wiki.go]: why `err != nil` but we return 200?

Comment (linquize) [inline routers/repo/wiki.go]: Commit updated @ 52e53ea.
I check if Home page exists first. If not exist, return 200
For GetWikiPage(), return 500 if error.

**Comment by linquize** [comment]
Updated commit @ e21eaf5

**Comment by lunny** [comment]
OKay. And any idea @tboerger ?

**Comment by tboerger** [comment]
Codewuse LGTM, but I have not tested it

**Pull request merged**